build.gradle

```diff
@@ -63,6 +63,11 @@ allprojects {
         google()
         mavenCentral()
     }
+    tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
```

Comment:
I think we just specify `KotlinCompile` instead `org.jetbrains.kotlin.gradle.tasks.KotlinCompile` right?

Reply:
I tried using `KotlinCompile` earlier but it threw an error.

```
A problem occurred evaluating root project 'News'.
> Could not get unknown property 'KotlinCompile' for root project 'News' of type org.gradle.api.Project.
```